web_monitoring/cli.py

```diff
@@ -26,21 +63,67 @@ def _add_and_monitor(versions):
         print("Errors: {}".format(errors))
 
 
-def import_ia(url, *, from_date=None, to_date=None, maintainers=None,
-              tags=None, skip_unchanged='resolved-response'):
+def import_ia_db_urls(*, from_date=None, to_date=None, maintainers=None,
+                      tags=None, skip_unchanged='resolved-response'):
+    client = db.Client.from_env()
+    logger.info('Loading known pages from web-monitoring-db instance...')
+    domains, version_filter = _get_db_page_url_info(client)
+    print('Importing {} Domains:\n  {}'.format(
+        len(domains),
+        '\n  '.join(domains)))
+
+    return import_ia_urls(
+        urls=[f'http://{domain}/*' for domain in domains],
+        from_date=from_date,
+        to_date=to_date,
+        maintainers=maintainers,
+        tags=tags,
+        skip_unchanged=skip_unchanged,
+        version_filter=version_filter)
+
+
+def import_ia_urls(urls, *, from_date=None, to_date=None, maintainers=None,
+                   tags=None, skip_unchanged='resolved-response',
+                   version_filter=None):
     skip_responses = skip_unchanged == 'response'
     with ia.WaybackClient() as wayback:
-        # Pulling on this generator does the work.
-        versions = (wayback.timestamped_uri_to_version(version.date,
-                                                       version.raw_url,
-                                                       url=version.url,
-                                                       maintainers=maintainers,
-                                                       tags=tags,
-                                                       view_url=version.view_url)
-                    for version in wayback.list_versions(url,
-                                                         from_date=from_date,
-                                                         to_date=to_date,
-                                                         skip_repeats=skip_responses))
+        def timestamped_uri_to_versions(ia_versions):
+            wayback_errors = {'playback': [], 'missing': [], 'unknown': []}
+            for version in ia_versions:
+                try:
+                    yield wayback.timestamped_uri_to_version(version.date,
+                                                             version.raw_url,
+                                                             url=version.url,
+                                                             maintainers=maintainers,
+                                                             tags=tags,
+                                                             view_url=version.view_url)
+                except ia.MementoPlaybackError as error:
+                    wayback_errors['playback'].append(error)
+                    logger.info(f'  {error}')
```

Comment:
In contrast to my comment about `logger.exception`, _these_ messages make sense as INFO messages to me because they are tracking expected errors coming from a service that issues them frequently.